core/js/typespecs/functions.ts

```diff
@@ -211,12 +211,22 @@ export const invokeParent = z.union([
     ),
 ]);
 
+const fetchRowFieldsSchema = z.object({
+  object_type: z.enum(["project_logs", "experiment", "playground_logs"]),
```

Comment:
Maybe we can share this wit the parent object type. Like `spanParentObjectTypeSchema = ...`.